app/models/cart_item.rb

```diff
@@ -0,0 +1,17 @@
+class CartItem < ApplicationRecord
+  belongs_to :user
+  belongs_to :product
+
+  before_create :check_quantity
+
+  private
+
+  def check_quantity
+    specific_product = Product.find(self.product_id)
+    puts specific_product
```

Comment:
you can remove the puts

Reply:
I created an additional commit to remove the puts

https://github.com/eefborromeo/private-dermatology/pull/22/commits/4e18161324461113bc2aab818c903d902adb52d7